src/types/api/core.ts

```diff
@@ -1,3 +1,8 @@
-export type ErrorDto = {
+export interface ErrorDto {
   message: string
 }
+
+export interface PrivateFormErrorDto extends ErrorDto {
```

Comment:
it doesn't make a difference but for the record, it wasn't necessary to change this to an interface, since you can use `&` to achieve the same thing